strutil/strutil.go

```diff
@@ -380,3 +380,17 @@ func JoinNonEmpty(strs []string, sep string) string {
 	}
 	return strings.Join(nonEmpty, sep)
 }
+
+// SplitNonEmtpy splits in the same way as strings.Split, but removes
+// any empty string in the returned slice.
+func SplitNonEmtpy(str, sep string) []string {
```

Comment:
```suggestion
func SplitNonEmpty(str, sep string) []string {
```

btw. there's strings.Fields() which already does what you want https://pkg.go.dev/strings?utm_source=godoc#Fields

Reply:
And there's also strings.FieldsFunc(), should work with path separator as well.

Reply:
see https://go.dev/play/p/9zc9CzXO2KP

Reply:
That's nice, thanks! I've replaced with the similar `strings.FieldsFunc()` as the separator is in this case `/`.

Reply:
Oh, I saw late your comments mentioning FieldsFunc :slightly_smiling_face: